signing/secp256k1/api/src/main/java/tech/pegasys/signers/secp256k1/api/PublicKey.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.secp256k1.api;
+
+public interface PublicKey {
+
+  byte[] getValue();
```

Comment:
Bytes instead byte[] would be more convenient. It does however add a dependency on Tuweni but that might not be so bad

Reply:
API literally has 0 dependencies - I would MUCH prefer to use Bytes, but thought the dependency bit was nicer.